src/scss/components/_aside.scss

```diff
@@ -22,19 +22,19 @@ aside {
 				li {
 					left: rem(-2);
 					margin: 0 0 rem(23) 0;
-					padding-left: rem(23);
-					width: rem(151);
+					padding-left: rem(20);
+					width: rem(158);
 					word-wrap: break-word;
 
-					&:active {
+					&.active {
 						box-shadow: rem(5) 0 0 0 $green-dark inset;
-						padding-left: rem(25);
+						padding-left: rem(22);
 						position: relative;
 					}
 
 					a {
 						color: $navy-light;
-						font-size: rem(24) !important; // Enactors.css in infusion uses !important on this element
+						font-size: rem(18) !important; // Enactors.css in infusion uses !important on this element
```

Comment:
Yes, I made sure to check this before pushing and this is the desktop definition. The aside menu is only on desktop and iPad Pro screen sizes.

This line is not needed for UIO default settings but without this line if enhance inputs is turned on the font size of the aside menu will increase because of the `.fl-input-enhanced a { font-size: 125% !important; }` line in the `Enactors.css`. Therefore this line is still necessary.

Reply:
I see. Thanks for the explanation.